libs/ui/src/components/NeoIcon/NeoIcon.vue

```diff
@@ -2,7 +2,7 @@
   <o-icon
     :pack="pack || 'fas'"
     :icon="icon"
-    :size="size || 'small'"
+    :size="size || ''"
```

Comment:
```suggestion
    :size="size || 'default'"
```
https://github.com/oruga-ui/oruga/blob/4915c4dbcb1a6c2cd39bc7660d41916b330bc0cd/packages/oruga/src/utils/icons.js#L46